mxd/backend-service/src/main/java/org/eclipse/mxd/entity/Content.java

```diff
@@ -0,0 +1,95 @@
+
+/*******************************************************************************
+ * Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *    Contributors:Ravinder Kumar
+ *    Backend-API and implementation
+ *
+ ******************************************************************************/
+
+
+package org.eclipse.mxd.entity;
```

Comment:
package is wrong. must start with `org.eclipse.tractusx.`

Reply:
I have changed package org.eclipse.mxd to org.eclipse.tractusx as suggested

Reply:
nope, it should _start_ with `org.eclipse.tractusx.`. Meaning, the correct package decl would then be e.g. `org.eclipse.tractusx.mxd.entity` etc.